src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,27 @@
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  errorMessage: ErrorType | null;
+  setErrorMessage: (error: ErrorType | null) => void;
```

Comment:
You never set actual ErrorMessage from inside this component, so I suggest rename it into close or clear, etc

```suggestion
  clear: () => void;
```